app/config/collections.php

```diff
@@ -3591,6 +3603,13 @@
             ],
         ],
     ],
+], $commonCollections);
+
+$collections = [
```

Comment:
I was thinking, maybe it will be easier to read if we just add a property to each collection stating where it should go.

Reply:
Not sure I understand.